src/index.js

```diff
@@ -51,12 +53,13 @@ const ping = (options, shouldMute) => {
     // Fires all requests
     .then(response => {
       log(`Loading details of the latest ${options.limit} top stories`);
-
-      return fetchTopStoriesDetails(response, options.limit);
+      return fetchTopStoriesDetails(response);
     })
     // Returns a formatted array with the response request
     .then(response => {
-      return response.map(item => item.body);
+      const listItems = [];
```

Comment:
@rafaelrinaldi i got it! i will refactor the code i agree with your terms! Thanks!